examples/qwen_npu/main.cpp

```
@@ -1,210 +1,288 @@
+                              ["sequence"];
+  MLLM_INFO("Input tokens: {} tokens", raw_input_tokens.shape()[1]);
 
-  mllm::models::ARGenerationOutputPast inputs{{"sequence", mllm::Tensor::empty({1, 128}, mllm::kInt64, mllm::kCPU).alloc()}};
+  const int eos_token_id = 151645;
+  const int max_new_tokens = 512;
+
+  mllm::models::ARGenerationOutputPast past{{"sequence", raw_input_tokens}};
+  mllm::models::ARGenerationArgs args;
+  args["debug_layer_outputs"] = false;
+
+  bool enable_timing = std::getenv("MLLM_ENABLE_LAYER_TIMING") != nullptr;
```

Clang (14.0.6)

[error] variable 'enable_timing' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)